charts/flagsmith/templates/deployment-task-processor.yaml

```diff
@@ -0,0 +1,103 @@
+{{- if .Values.taskProcessor.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "flagsmith.fullname" . }}-task-processor
+  labels:
+    {{- include "flagsmith.labels" . | nindent 4 }}
+    app.kubernetes.io/component: task-processor
+spec:
+  selector:
+    matchLabels:
+      {{- include "flagsmith.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: task-processor
+  replicas: {{ .Values.taskProcessor.replicacount }}
+  template:
+    metadata:
+      annotations:
+        checksum/secrets-api: {{ include (print $.Template.BasePath "/secrets-api.yaml") . | sha256sum }}
+        {{- if and .Values.influxdbExternal.enabled (not .Values.influxdbExternal.tokenFromExistingSecret.enabled) }}
+        checksum/secrets-influxdb-external: {{ include (print $.Template.BasePath "/secrets-influxdb-external.yaml") . | sha256sum }}
+        {{- end }}
+{{- if .Values.taskProcessor.podAnnotations }}
+{{ toYaml .Values.taskProcessor.podAnnotations | nindent 8 }}
+{{- end }}
+      labels:
+        {{- include "flagsmith.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: task-processor
+        {{- if .Values.taskProcessor.podLabels }}
+{{ toYaml .Values.taskProcessor.podLabels | indent 8 }}
+        {{- end }}
+    spec:
+      {{- if .Values.taskProcessor.affinity }}
+      affinity:
+{{ toYaml .Values.taskProcessor.affinity | indent 8 }}
+      {{- end }}
+      {{- if .Values.taskProcessor.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.taskProcessor.nodeSelector | indent 8 }}
+      {{- end }}
+      {{- if .Values.taskProcessor.tolerations }}
+      tolerations:
+{{ toYaml .Values.taskProcessor.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.taskProcessor.schedulerName }}
+      schedulerName: "{{ .Values.taskProcessor.schedulerName }}"
+      {{- end }}
+      {{- if .Values.api.image.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.api.image.imagePullSecrets | indent 4 }}
```

Comment:
@plumdog based on the PR we received along a similar vein to this, should this be `indent 8` ?

Reply:
Good spot - agreed and fixed.